ci_framework/hooks/playbooks/operator_installation_check.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Checkout to OpenStack Namespace
+      ansible.builtin.command: oc project openstack
+      changed_when: false
+
+    - name: Make sure all Openstack operators are deployed
+      ansible.builtin.shell: |
+        set -o pipefail && oc get csv -l operators.coreos.com/openstack-operator.openstack --no-headers=true | grep -i "succeeded"
```

Comment:
We can iterate on that, we can really leverage it using pure ansible with a simple "command" call imho.
As a follow-up though. Please add a TODO for later reference? :)

Reply:
Why are we needing this? The framework itself is waiting for openstack metaoperator it-self to be up.

Reply:
Removed it.